libs/application-generic/src/usecases/upsert-preferences/upsert-preferences.usecase.ts

```diff
@@ -35,6 +43,32 @@ export class UpsertPreferences {
     });
   }
 
+  private async deleteSubscriberChannelPreferences(
```

Comment:
This method is responsible for "forgetting" the subscriber workflow channel preferences when the subscriber modifies a global preference for 1 or more channels.